js/foundation/foundation.topbar.js

```diff
@@ -258,6 +262,10 @@
         }
 
         S('[' + self.attr_name() + '] li.hover').removeClass('hover');
+        
+        if(!settings.is_hover){
```

Comment:
settings.is_hover returns undefined, please add in settings to the scope, IE line 258 should now be 
```
var parent = S(e.target).closest('li').closest('li.hover'),
topbar = S(e.target).closest('[' + self.attr_name() + ']'),
settings = topbar.data(self.attr_name(true) + '-init');
```